test/lib/pipelineFactory.test.js

```diff
@@ -85,8 +85,7 @@ describe('Pipeline Factory', () => {
                 data: {
                     admins,
                     createTime: nowTime,
-                    scmUrl,
-                    configUrl: scmUrl
+                    scmUrl
```

Comment:
Should also remove `configUrl` here
https://github.com/screwdriver-cd/models/blob/9225a81c0cd949d9f80964a16bbc43ed10626ef8/test/lib/pipelineFactory.test.js#L53